docs/MongoDB-batchsink.md

```diff
@@ -0,0 +1,32 @@
+# MongoDB Batch Sink
```

Comment:
I think you miss icons for both plugins

Reply:
It seems that the core [has this icon built-in](https://github.com/cdapio/cdap/blob/develop/cdap-ui/app/directives/dag-plus/my-dag-factory.js#L191).